src/test/java/secretclosetapp/utils/Constants.java

```diff
@@ -0,0 +1,6 @@
+package secretclosetapp.utils;
```

Comment:
utils package is only for util classes, but yours can be moved to **constants** package